src/inlineImages.js

```diff
@@ -4,13 +4,13 @@ const fs = require('fs').promises;
 const path = require('path');
 
 let inlineImages = async htmlPath => {
-	const imgTagRegex = /<img (.* )?src="([\w.\-\/]+)"(.*)>/;
+	const imgTagRegex = /<img (.* )?src="?([\w.\-\/]+)"?(.*)>/;
```

Comment:
If we're going to relax this, then we may as well include single quotes as well.
Specifically, here and in the other files, replace `"?` with `['"]?`

Reply:
Hey @mahhov, sorry I didn't get back to you on this. I think my change actually breaks in some cases (can't recall the exact problem now) - it happened to work fine on my minified stylesheet, but didn't really test other cases. Merging this PR at the current state is probably not a good idea, unless I dig into the problem and add some more tests!